README.md

```diff
@@ -404,33 +404,89 @@ With the CodePush plugin downloaded and linked, and your app asking CodePush whe
 
 2. When an update is available, how to present it to the end user?
 
-The simplest way to do this is to perform the following in your app's root component:
+The simplest way to do this is to "CodePush-ify" your app's root component. To do so, you can choose one of the following three options:
 
-1. Import the JavaScript module for CodePush:
+**Option 1: Wrap your root component using the `codePush` decorator function:**
+
+```javascript
+import codePush from "react-native-code-push";
+
+let codePushOptions;
+
+class MyApp extends Component {
+}
+
+MyApp = codePush(codePushOptions)(MyApp);
+```
+
+**Option 2: Use the [ES7 decorator](https://github.com/wycats/javascript-decorators) syntax:**
+
+```javascript
+import codePush from "react-native-code-push";
+
+let codePushOptions;
+
+@codePush(codePushOptions)
+class MyApp extends Component {
+}
+```
+
+*NOTE: Decorators are not yet supported in Babel 6.x pending proposal update.* You may need to enable it by doing the following:
+- Install the [`babel-preset-react-native-stage-0` package](https://github.com/skevy/babel-preset-react-native-stage-0)
```

Comment:
To avoid cluttering the getting started, could we take these babel instructions and move them to a separate section or README and just link to them here?